### DISCUSSION TITLE
New plugin not recognized

### DISCUSSION THREAD

**Discussion by fspinar** [jspsych/jsPsych#2460] [Q&A]
Hi all,

I am trying to create and use a new plugin for my project. But I get the following error at the webpage: 
No plugin loaded for trials of type "my plugin name"

I am pretty sure that I don't do a typo. Is there anything special that I need to do after writing the code for the plugin in order for application to recognize it ?

PS : I am implementing my project on Psiturk.

Thanks !

**Comment by jodeleeuw** [comment]
Hi @fspinar,

Would you please share the plugin code and the experiment code? This is hard to diagnose without seeing those files.

**Comment by becky-gilbert** [comment]
Hi @fspinar, it sounds like you're using jsPsych v7, right? Can you say a bit about what files you've added/edited to create your plugin? Are you using a typescript plugin template and build steps, or directly editing a JavaScript file?
